test/e2e/kn_admin_test.go

```diff
@@ -151,6 +153,22 @@ func (et *e2eTest) knAdminAutoscaling(t *testing.T, r *test.KnRunResultCollector
 	r.AssertNoError(out)
 	out = et.kn.Run(pluginName, "autoscaling", "update", "--scale-to-zero")
 	r.AssertNoError(out)
+	out = et.kn.Run(pluginName, "autoscaling", "list")
```

Comment:
I would just reuse the constant defined in code to list through all the config names

Reply:
ok, I will check all config names and values.